Add a hint about composer cache clean on the Cloud environment (#8890)
magento/devdocs: src/cloud/setup/first-time-setup-import-import.md

After the [cache](https://glossary.magento.com/cache) flushes, enter `exit` to close the SSH tunnel.

> {:.bs-callout-info}
> To clear composer cache on Cloud environment use [magento-cloud project:clear-build-cache]({{ site.baseurl }}/guides/v2.4/reference/cli/magento-cloud.html) command.
> 

Reviewer: This would be a nice tip.

Now, is this a composer cache or a build cache or both? For composer I would think you would use `composer cc`, but then that won't clean anything in a remote Cloud environment. I made a suggestion, but it may not be quite what you wanted to say.

```suggestion
{:.bs-callout-tip}
Use the [magento-cloud project:clear-build-cache]({{ site.baseurl }}/guides/v2.4/reference/cli/magento-cloud.html) command to clear project build artifacts in the remote environment.

```

Reviewer 2: @EvheniiaKolkhiieva , could  you please add `#projectclear-build-cache` to link:

```
[magento-cloud project:clear-build-cache]({{ site.baseurl }}/guides/v2.4/reference/cli/magento-cloud.html#projectclear-build-cache)
```

Reviewer 2: Hi, @hguthrie!

I would like a bit extend your option to regarding to [Platform.sh CLI](https://github.com/platformsh/platformsh-cli) terminology. 

**My option is:** Use the `magento-cloud project:clear-build-cache` command to clear a project's build cache (such as composer cache, etc) in the remote environment.

Author: Hi @hguthrie and @atwixfirster, thank you!

I have implemented your suggestion, could you please check it and approve?

> To clear composer cache on Cloud environment use [magento-cloud project:clear-build-cache](https://devdocs.magento.com/guides/v2.4/reference/cli/magento-cloud.html) command.

Reviewer 3: Please use the liquid variables for versioned devdocs URLs.

Author: @dobooth done, please check.

thank you!
